### DISCUSSION TITLE
Ignore .file.swp

### DISCUSSION THREAD

**Discussion by johnununu** [Security-Onion-Solutions/securityonion#7390] [Unsupported Versions] [answered]
Is it possible to ignore swap files in salt?

While I'm working on files in /opt/so/saltstack/local/salt/elasticsearch/files/ingest with vim, the .swp is created.
Prior to saving my work salt detects the addition of the .swp file and does it's thing.

'X' of 600 shards failed starts flying on the right side of the dashboard.

**Comment by dougburks** [comment, marked as answer]
You could just disable swap files in vim:
https://stackoverflow.com/questions/821902/disabling-swap-files-creation-in-vim
